compiler/src/dotty/tools/repl/ReplDriver.scala

```diff
@@ -323,7 +323,7 @@ class ReplDriver(settings: Array[String],
         .find(_.symbol.name == newestWrapper.moduleClassName)
         .map { wrapperModule =>
           val formattedTypeDefs = typeDefs(wrapperModule.symbol)
-          val (newState, formattedMembers) = displayMembers(wrapperModule.symbol)
+          val (newState, formattedMembers) = extractAndFormat(wrapperModule.symbol)
```

Comment:
typo:
```suggestion
          val (newState, formattedMembers) = extractAndFormatMembers(wrapperModule.symbol)
```

Reply:
You are too fast -- I already changed it locally but haven't pushed yet :)